frontend/src/scenes/experiments/ExperimentPreview.tsx

```diff
@@ -444,7 +444,7 @@ export function MetricDisplay({ filters }: { filters?: FilterType }): JSX.Elemen
                 .map((event: ActionFilterType, idx: number) => (
                     <div key={idx} className="mb-2">
                         <div className="flex mb-1">
-                            <div className="preview-conversion-goal-num">
```

Comment:
oh, sounds like you can delete this class?

Reply:
Indeed